lib/engine.js

```diff
@@ -102,7 +102,10 @@ class Engine {
      * @returns 
      */
     wrapError(sourceError, message, location) {
-        if (!(sourceError instanceof GenericError) && !(sourceError instanceof ClientError)) {
+        if (!(sourceError instanceof GenericError) 
+            && !(sourceError instanceof ClientError)
+            && !(sourceError instanceof RenditionFormatUnsupportedError)
+            && !(sourceError instanceof RenditionTooLarge)) {
```

Comment:
insteresting the `ClientError` did not cover instanceof `RenditionFormatUnsupportedError` and `RenditionTooLarge` since they are client errors: https://github.com/adobe/asset-compute-commons/blob/master/lib/errors.js#L116

with this logic, you would need to add the other client errors to the list:  `SourceFormatUnsupportedError`, `SourceUnsupportedError` and `SourceCorruptError`

Reply:
Do we want them here too? 
I wasn't sure if Source* errors can even happen here.